src/schema/budget-line.ts

```diff
@@ -0,0 +1,14 @@
+import { pgTable, text } from "drizzle-orm/pg-core";
+import { z } from "zod";
+
+export const budgetLine = pgTable("budget_line", {
+  code: text("code"),
+  id: text("id"),
```

Comment:
should have a [composite primary key](https://orm.drizzle.team/docs/indexes-constraints#composite-primary-key) with code and id.

Reply:
This will has the added complication of 'code' itself being a foreign key to the agency_budget.code column. Let me know if added this foreign key gives you trouble.